src/Ignitor/SignalsListener/index.ts

```diff
@@ -19,6 +19,7 @@ export class SignalsListener {
    */
   private kill = async function () {
     try {
+      console.log('Shutting down server...')
```

Comment:
Modules should never have hardcoded console statements

Reply:
Hmm, should use debugger instead?